content/blog/anirudhm/2017-05-21_[gsoc_2017]_preferences_gui_refactoring_-_weekly_report_1.md

```diff
@@ -0,0 +1,30 @@
++++
+type = "blog"
+title = "[GSoC 2017] Preferences GUI Refactoring - Weekly Report 1"
+author = "anirudhm"
+date = "2017-05-21 01:32:01+05:30"
+tags = ["haiku", "gsoc", "gsoc2017", "preferences"]
++++
+
+Hello World.
+
+This is an update to my previous blog post which dealt with the introduction to the GSoC project which I’ll be working this summer - *Preferences GUI Refactoring*. It's been two weeks since the first post went live, so here's the report for the weeks after that.
+This being the *Community Bonding* period, I pretty much did what the title says. Got to know about fellow GSoCers, they really are friendly and helpful. Myself, my mentors (waddlesplash and Sean), PulkoMandy, jua_ and humdinger had our first meeting, infact a very long meeting. The meeting was more of a discussion on the last blog post's comments. Last blog post received so many suggestions and constructive criticisms, let me thank everyone for their valuable feedback. We did take everything into concern. The major changes that are done to the project after the blog post:
+
+The control panel (we call it **SuperPrefs** for now) will be segregating preference panels category wise, along with an inner search implementation (as explained in last blog post). We're dropping off the _Preview pane_, at least for now, it doesn't seem to make many changes and there were more major things to be addressed. Incase you missed the last post, here’s link to [that](https://www.haiku-os.org/blog/anirudhm/_gsoc_2017_preferences_gui_refactoring_intro/).
```

Comment:
Should probably note that SuperPrefs is not the final name; the final name will probably just be "Preferences" or the like